posthog/templates/demo.html

```diff
@@ -157,5 +163,12 @@ <h2>Reviews</h2>
 
 </div>
 
+    <script>
```

Comment:
Let's remove changes on this file: master should now use the latest posthog.js which achieves the same.